src/main/java/org/sagebionetworks/web/client/widget/table/modal/fileview/CreateTableViewWizardStep1.java

```diff
@@ -134,8 +135,14 @@ private void createEntity(final Entity entity) {
 		jsClient.createEntity(entity).addCallback(new FutureCallback<Entity>() {
 			@Override
 			public void onSuccess(Entity table) {
-				step2.configure((Table) table, tableType);
-				modalPresenter.setNextActivePage(step2);
+				// For Datasets, this is the only step.
+				if (TableType.dataset.equals(tableType)) {
+					modalPresenter.onFinished();
```

Comment:
Test in CreateTableViewWizardStep1Test to verify early exit for Datasets?

Reply:
https://github.com/Sage-Bionetworks/SynapseWebClient/pull/4709/files#diff-f0702dba294bd57c898b4bd1693895c233db7ed5bb5063042687fd5651298f22R217